plugins/modules/aci_l3out_bfd_interface_profile.py

```diff
@@ -0,0 +1,353 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Anvitha Jain (@anvjain)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_bfd_interface_profile
+short_description: Manage L3Out BFD Interface profile.
```

Comment:
Should we combine this one with the multihop one? All the input is basically the same, only the classnames differ.

Reply:
As discussed, since the module is already ready we will not be merging them.